pubspec.yaml

```diff
@@ -34,5 +35,17 @@ flutter:
   fonts:
     - family: Inter
       fonts:
-        - asset: assets/fonts/Inter.ttf
+        - asset: assets/fonts/Inter-300-Light.ttf
```

Comment:
nit: consider using package:google_fonts instead of hard including inter.

More detail: https://pub.dev/packages/google_fonts

Reply:
Downside of google fonts is it requires http download one time before fonts load. Not a fan of that.

Reply:
Fair enough. Have you raised a bug on `google_fonts` about the http download requirement? I'll happily advocate for a build time option with the eng team.

Reply:
I haven't raised it. I assumed it was WAI as it gives you easy access to the online library of Google fonts (major plus) with the downside it needs to do a small download upon startup. Using local assets seems to be a fine alternative if you just want a small number of fonts with no http download (downside being increased binary size).

Reply:
Fair. I'll still bug the team tho =)